distutils/command/build_py.py

```diff
@@ -360,7 +360,7 @@ def build_modules(self) -> None:
             self.build_module(module, module_file, package)
 
     def build_packages(self) -> None:
-        for package in self.packages:
+        for package in self.packages or ():
```

Comment:
This is a change in behaviour as it would graciously do nothing instead of raising if `self.packages` is still `None`.

Reply:
Feel free to update the message to your likings !